cpu/saml21/periph/spi.c

```diff
@@ -97,6 +97,8 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             f_baud = 1000000;
             break;
+        case SPI_SPEED_4MHZ:
+            return -1;
```

Comment:
I think we should get this PR in shape now and wait for #4780 with the actual driver, if you don't have any time pressure.